### PR TITLE
Add Azure synapse feature in what's new page

### PR DESCRIPTION
**Pull request by t-rufang**

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/resources/whatsnew/whatsnew.md [hunk 1]
```diff
@@ -1,10 +1,34 @@
-<!-- Version: 3.37.0 -->
+<!-- Version: 3.38.0 -->
 # What's new in Azure Toolkit for IntelliJ
 
 > Scroll down to checkout our newly added support for **Azure Functions**, **Azure Spring Cloud** and
  **log streaming for Azure App Service**.
 
-## 3.37.0(Current Release)
+## 3.38.0(Current Release)
+We are proud to announce the support for [Azure Synapse](https://docs.microsoft.com/en-us/azure/synapse-analytics/spark/intellij-tool-synapse) in Azure toolkit! The following feature is waiting to be discovered.
```
Comment (Flanker32) [inline PluginsAndFeatures/azure-toolkit-for-intellij/resources/whatsnew/whatsnew.md:8]: following feature is -> following features are ?

File: PluginsAndFeatures/azure-toolkit-for-intellij/resources/META-INF/plugin.xml [hunk 2]
```diff
@@ -6,14 +6,15 @@
 
   <description><![CDATA[
       <html>
-        <p>These plugins allow Java developers, Azure HDInsight developers and Apache Spark on SQL Server users to easily create, develop, configure, test, and deploy highly available and scalable Java web apps and Apache Spark/Hadoop jobs to Azure from IntelliJ on all supported platforms.</p>
+        <p>The plugin allows Java developers to easily develop, configure, test, and deploy highly available and scalable Java web apps. It also supports Azure Synapse data engineers, Azure HDInsight developers and Apache Spark on SQL Server users to create, test and deploy Apache Spark/Hadoop jobs to Azure from IntelliJ on all supported platforms.</p>
```
Comment (Flanker32) [inline PluginsAndFeatures/azure-toolkit-for-intellij/resources/META-INF/plugin.xml:9]: deploy Apache Spark/Hadoop jobs -> submit Apache Spark/Hadoop jobs ?

Comment (t-rufang) [reply PluginsAndFeatures/azure-toolkit-for-intellij/resources/META-INF/plugin.xml:9]: Good catch! Will fix it in next commit.

File: PluginsAndFeatures/azure-toolkit-for-intellij/resources/whatsnew/whatsnew.md (hunk 1 above)
Comment (t-rufang) [reply PluginsAndFeatures/azure-toolkit-for-intellij/resources/whatsnew/whatsnew.md:8]: Good catch!

**Pull request merged**